### PR TITLE
Handlebar/games

### PR DESCRIPTION
**Pull request by KorolyovaLara**

### CODE REVIEW

**Comment by KorolyovaLara** [comment]
DON'T MERGE IT, still in progress

File: routes/html/index.js [hunk 1]
```diff
@@ -47,8 +48,13 @@ router.get("/user/:username", async (req, res) => {
   res.render("user-info", { games, 404: false, username });
 });
 
-router.get("/games", (req, res) => {
-  res.render("games", {});
+// get all games page
+router.get("/games", async (req, res) => {
+  const gamesData = await Games.findAll().catch((err) => {
```
Comment (strahinjaajvaz) [inline routes/html/index.js]: Use try catch

File: server.js [hunk 2]
```diff
@@ -29,9 +29,15 @@ app.use(setNewToken);
 
 // 404 page
 app.use(function (req, res) {
-  res.render("404", { title: "Oops! Page not found." });
+  res.render("404", {
+    title: "Oops! Page not found.",
+    partials: "404",
```
Comment (strahinjaajvaz) [inline server.js]: What's with this?

**Review by strahinjaajvaz — CHANGES_REQUESTED** [review]

File: server.js (hunk 2 above)
Comment (KorolyovaLara) [reply server.js]: that's called - Lara was lost, but then reworked it all

File: routes/html/index.js (hunk 1 above)
Comment (KorolyovaLara) [reply routes/html/index.js]: Done

File: views/games.handlebars [hunk 3]
```diff
@@ -1 +1,20 @@
...
+        <div class="title level-item">
+            <h1> All Games that our Users Play </h1>
+        </div>
+        {{#each games as |game|}}
+            <div class="hero-body">
+                <p class="level-item">{{game.title}}.</p>
+                <p class="level-item">{{game.description}}.</p>
+
+                <figure class="image is-16by9">
+                    <iframe class="has-ratio" width="640" height="360" src="{{game.trailer}}" frameborder="0" allowfullscreen  alt="video for {{game.title}} game"></iframe>
```
Comment (strahinjaajvaz) [inline views/games.handlebars]: Your alt doesn't make sense.
```suggestion
                    <iframe class="has-ratio" width="640" height="360" src="{{game.trailer}}" frameborder="0" allowfullscreen  alt="trailer for {{game.title}}."></iframe>
```

**Review by tinku-bansal — APPROVED** [review]
Looks Good!

File: views/games.handlebars (hunk 3 above)
Comment (KorolyovaLara) [reply views/games.handlebars]: @strahinjaajvaz ok understood

**Pull request merged**